src/pages/app-about/app-about.tsx

```diff
@@ -10,6 +10,9 @@ declare var fbq;
 export class AppAbout {
   @Prop({ context: 'isServer' })
   private isServer: boolean;
+  private className = localStorage.getItem('allowWebp') === 'true' ? 'webp' : 'hero';
```

Comment:
You forgot here, on about page add the class webp to use the webp image as in the other pages